scripts/e2e/localnet.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/sh
+set -e
+
+# Console log text colour
```

Comment:
Why use a separate localnet.sh instead of just using [scripts/localnet.sh](https://github.com/NibiruChain/nibiru/blob/master/scripts/localnet.sh) ?

The one in this PR is outdated.

Reply:
just for the nibid binary most likely I did it